code_schemes/kakuma_s01e06.json

```diff
@@ -84,6 +84,16 @@
       "StringValue": "CE",
       "VisibleInCoda": false
     },
+    {
+      "CodeID": "code-DNS-982f3ca2",
+      "CodeType": "Control",
+      "ControlCode": "DNS",
+      "DisplayText": "DNS (do not share)",
```

Comment:
Over in the JPLG project you had  some suggestions for alternative names to use for this. Last chance to push for an alternative if there's one you prefer!

Reply:
do not share is super clear to the RAs - so lets keep it.